tools/rgrnastar/rg_rnaStarSolo.xml

```diff
@@ -101,7 +101,7 @@
     --quantMode TranscriptomeSAM $solo.quantModeGene
     #set $tag_names = str($solo.outSAMattributes).replace(',', ' ')
     --outSAMattributes $tag_names
-    #if "CB" in $tag_names or "UB" in $tag_names:
+    #if "CB" in $tag_names or "UB" in $tag_names or str($coverage.outWig.outWigType) != 'None':
         --outSAMtype BAM SortedByCoordinate
     #else:
         --outSAMtype BAM Unsorted
```

Comment:
The format of the bam output should be set to `unsorted.bam` in this case, or?

Reply:
No below there are samtools command that sort it.

Reply:
Here is the line: https://github.com/lldelisle/tools-iuc/blob/9c5210c19fda711a9fb9af1cc2aed425a3c04512/tools/rgrnastar/rg_rnaStarSolo.xml#L150